docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sort/search-sorted-15.rst

```diff
@@ -0,0 +1,72 @@
+SearchSorted
+===============
+
+
+.. meta::
+  :description: Learn about SearchSorted - a sorting and maximization
+                operation, which requires two input tensors.
+
+
+**Versioned name**: *SearchSorted-15*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: Find the indices from the innermost dimension of sorted_sequence such that, if the corresponding values in values were inserted before the indices, when sorted, the order of the corresponding innermost dimension within sorted_sequence would be preserved.
+
+**Detailed description**: `Reference <https://pytorch.org/docs/stable/generated/torch.searchsorted.html>`__.
+
+**Attributes**
+
+* *right*
+  * **Description**: *right*  if False, return the first suitable location that is found. If True, return the last such index. If no suitable index found, return 0 for non-numerical value (eg. nan, inf) or the size of innermost dimension within sorted_sequence (one pass the last index of the innermost dimension). In other words, if False, gets the lower bound index for each value in values on the corresponding innermost dimension of the sorted_sequence. If True, gets the upper bound index instead. Default value is False. side does the same and is preferred. It will error if side is set to “left” while this is True.
```

Comment:
```suggestion
  * **Description**: *right*  if False, return the first suitable location that is found. If True, return the last such index. If no suitable index is found, return 0 for non-numerical value (e.g. nan, inf) or the size of innermost dimension within sorted_sequence (one pass the last index of the innermost dimension). In other words, if False, get the lower bound index for each value in values on the corresponding innermost dimension of the sorted_sequence. If True, get the upper bound index instead. Default value is False. Side does the same and is preferred. It will error if side is set to “left” while this is True.
```